internal/cmd/server.go

```diff
@@ -77,7 +83,11 @@ var ServerCmd = &cobra.Command{
 			log.Printf("Done running powfile: %q\n", powfile)
 		}
 
-		select {}
+		if debug {
+			processLogs()
```

Comment:
The live cicle of the server depends how's logging?

Reply:
No, depending on how the server is started the mechanism to maintain the main gorutine is different. If debugging mode is active the main thread is waiting for messages if uot debugging must wait in an infinite select because it has nothing to do